example/lib/main.dart

```diff
@@ -58,6 +59,7 @@ class _MyHomePageState extends State<MyHomePage> {
             },
             onCompleted: (pin) {
               print("Completed: " + pin);
+              otpController.setFocus(4);
```

Comment:
after all of the boxes have added text why are you setting the focus on 4th box?